Netflix Movies/Netflix Movies/Controllers/Core/HomeViewController.swift

```diff
@@ -17,51 +17,76 @@ enum Sections: Int {
 
 final class HomeViewController: UIViewController  {
     
-    let sectionTitles: [String] = ["Trending Movies", "Trending TV",  "Popular", "Upcomming Movies", "Top Raced"]
-    let numberOfSelection = 40
-    let numberHeightForRowAt = 200
-    let titleRepository = TitleRepository()
-    var listTitle = [Title]()
+    private let sectionTitles: [String] = ["Trending Movies", "Trending TV",  "Popular", "Upcomming Movies", "Top Raced"]
+    private let numberOfSelection = 40
+    private let numberHeightForRowAt = 200
+    private let titleRepository = TitleRepository()
+    private var listTitle = [Title]()
     private var titleInfo: Title?
     private let network = APICaller.shared
+    private let numberOfRowsInSection = 1
+    private var titles: [Title] = [Title]()
```

Comment:
sao có 2 cái titles với listTitle vậy bro :v
```suggestion
    private var titles = [Title]()
```